File: app/src/main/java/intellinectsschool/intellinects/com/v4flavors/Fragments/SwipeFragment.java
package intellinectsschool.intellinects.com.v4flavors.Fragments;


import android.app.Fragment;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import intellinectsschool.intellinects.com.v4flavors.R;


/**
 * A simple {@link Fragment} subclass.
 */
public class SwipeFragment extends Fragment {

    public SwipeFragment() {
        // Required empty public constructor
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_swipe, container, false);
    }
}

File: app/src/main/java/intellinectsschool/intellinects/com/v4flavors/Fragments/SwipeFragment.kt
package intellinectsschool.intellinects.com.v4flavors.Fragments


import android.app.Fragment
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import intellinectsschool.intellinects.com.v4flavors.R


/**
 * A simple [Fragment] subclass.
 */
class SwipeFragment : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_swipe, container, false)
    }
}// Required empty public constructor
